src/Symfony/Component/DependencyInjection/Container.php

```diff
@@ -219,7 +219,7 @@ public function set($id, $service, $scope = self::SCOPE_CONTAINER)
      */
     public function has($id)
     {
-        for ($i = 2;;) {
+        for ($i = 2; ;) {
```

Comment:
Please be aware that the fixer `space_after_semicolon` @ Symfony set does this by design (https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/2.3/tests/Fixer/Semicolon/SpaceAfterSemicolonFixerTest.php#L87)
So this might come back in other PR's. Just saying if this is a SF rule this might be worth to change in the fixer project in some way.